srcpkgs/GraphicsMagick/template

```diff
@@ -45,7 +45,7 @@ libgraphicsmagick-devel_package() {
 		vmove "usr/lib/*.so"
 		vmove "usr/lib/*.la"
 		vmove "usr/share/man/man1/*-config.1"
-		vcopy ${wrksrc}/PerlMagick usr/share/doc/GraphicsMagick*/
+		vcopy ${wrksrc}/PerlMagick usr/share/doc/GraphicsMagick/
```

Comment:
```suggestion
		vcopy PerlMagick usr/share/doc/GraphicsMagick/
```
PWD should be $wrksrc already

Reply:
Good catch, done. Only change in the force-push.